### PR TITLE
update gdbserver

### PR DESCRIPTION
**Pull request by kabeor**

### CODE REVIEW

File: docs/GDBSERVER.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ from qiling import *
 
 def test_gdb(path, rootfs):
     ql = Qiling(path, rootfs, output="off")
-    # ql.gdb = None  # disable gdbserver
```
Comment (aquynh) [inline docs/GDBSERVER.md]: dont delete this line, you can replace it with: 

`# ql.gdb = True  # enable gdbserver at default port`

**Comment by xwings** [comment]
thanks !

File: qiling/core.py [hunk 2]
```diff
@@ -208,10 +208,26 @@ def shellcode(self):
...
     def run(self):
+        if self.gdb is not None:
+            try:
+                if self.gdb is True:
+                    self.gdbserver()
+                else:
+
+                        ip, port = self.gdb.split(':')
+                        port = int(port)
+                        self.gdbserver(ip, port)
```
Comment (cq674350529) [inline qiling/core.py]: incorrect indention and extra empty line :(

File: qiling/core.py [hunk 3]
```diff
@@ -208,10 +208,26 @@ def shellcode(self):
 
 
     def run(self):
+        if self.gdb is not None:
+            try:
+                if self.gdb is True:
+                    self.gdbserver()
+                else:
+
+                        ip, port = self.gdb.split(':')
```
Comment (cq674350529) [inline qiling/core.py]: incorrect indention and extra empty line :(

Comment (kabeor) [reply qiling/core.py]: oh，my bad

Comment (kabeor) [reply qiling/core.py]: will PR soon, thanks!

**Pull request merged**